Nef_3/include/CGAL/Nef_3/SNC_point_locator.h

```diff
@@ -223,220 +230,203 @@ class SNC_point_locator_by_spatial_subdivision :
     return this->shoot(ray, null_handle, mask);
   }
 
+  enum SOLUTION { is_vertex_, is_edge_, is_facet_ };
```

Comment:
We have a [`Feature_dimension` ](https://doc.cgal.org/latest/Kernel_23/structCGAL_1_1Feature__dimension.html) class.   Even if it is maybe not the perfect fit here, it might lead to  a better name than `SOLUTION`.  And note  that all uppercase is for constants. Types are just capitalized.

Reply:
Ok I'll look into it. It was copied from `SM_point_locator`. I like to make things consistently inconsistent :joy:

Reply:
I'll leave this suggestion for potential refactoring in the future.